packages/node_modules/@webex/internal-plugin-ediscovery/README.md

```diff
@@ -31,6 +31,12 @@ spark.internal.plugin.ediscovery.WHATEVER
 
 ```
 
+## Tests
+
+Run the ediscovery from the top level using 
+
+> npm test -- --package @webex/internal-plugin-ediscovery --node
```

Comment:
Not sure if this is a good practice to list here. But I need to store that command somewhere sensible. Otherwise it'll be forgotten.

Reply:
I think we should include more examples actually.